pages/evaluation-form/index.js

```diff
@@ -0,0 +1,50 @@
+import EmojiButton from "../../components/EmojiButton";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+export default function EvaluationForm({ moods, setMoods }) {
+  const router = useRouter();
+  const [mood, setMood] = useState("");
+  const [body, setBody] = useState("");
+
+  function handleSave() {
+    const currentDate = new Date().toLocaleDateString();
+    setMoods([...moods, { mood, body, date: currentDate }]);
+    router.push(`/progress-list`);
+  }
+
+  function handleMoodChange(newMood) {
+    setMood(newMood);
+  }
+
+  function handleBodyChange(newBody) {
+    setBody(newBody);
+  }
+
+  return (
+    <>
+      <h2>Wie ist deine Stimmung heute?</h2>
+      <EmojiButton emoji="😤" onChange={handleMoodChange} />
+      <EmojiButton emoji="😭" onChange={handleMoodChange} />
+      <EmojiButton emoji="😩" onChange={handleMoodChange} />
+      <EmojiButton emoji="☹️" onChange={handleMoodChange} />
+      <EmojiButton emoji="😕" onChange={handleMoodChange} />
+      <EmojiButton emoji="🙂" onChange={handleMoodChange} />
+      <EmojiButton emoji="😊" onChange={handleMoodChange} />
+      <EmojiButton emoji="😁" onChange={handleMoodChange} />
+      <EmojiButton emoji="🥳" onChange={handleMoodChange} />
+
+      <h2>Wie fühlt sich dein Körper?</h2>
+      <EmojiButton emoji="😤" onChange={handleBodyChange} />
+      <EmojiButton emoji="😭" onChange={handleBodyChange} />
+      <EmojiButton emoji="😩" onChange={handleBodyChange} />
+      <EmojiButton emoji="☹️" onChange={handleBodyChange} />
+      <EmojiButton emoji="😕" onChange={handleBodyChange} />
+      <EmojiButton emoji="🙂" onChange={handleBodyChange} />
+      <EmojiButton emoji="😊" onChange={handleBodyChange} />
+      <EmojiButton emoji="😁" onChange={handleBodyChange} />
+      <EmojiButton emoji="🥳" onChange={handleBodyChange} />
```

Comment:
you could also do this with a map over emojis.

Reply:
Good idea. Thanks  😊  I will try to implement it in one of the next PRs